github-templates/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+(brief description / PR title)
+
+Status: **Ready for Review** or **Open for Visibility**
```

Comment:
@marlowpayne any chance you'll have some time to take a swing at this? Would love to get this merged!